.rubocop.yml

```diff
@@ -215,6 +215,46 @@ Lint/UselessAssignment:
 Lint/DeprecatedClassMethods:
   Enabled: true
 
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: true
```

Comment:
This is very useful! :muscle:  I still hold hope for ruby/openssl#377. :crossed_fingers:

Reply:
I do prefer to disscuss about it in #39410, #39504, #39540.